.github/workflows/integration-test-mgr-suite.yaml

```diff
@@ -1,6 +1,7 @@
 name: Integration test CephMgrSuite
 on:
-  pull_request:
+  schedule:
+    - cron: "0 0 * * MON" # every monday at midnight
```

Comment:
why not run it daily?

Reply:
I thought we agreed on weekly, which seems to be the pace of code changes in ceph master.

Reply:
It could be helpful to run it daily just to see which day of the week anything changed. Also, for verifying changes we should probably keep this test on PRs at least when a label is added to the PR.